frontend/src/components/sendRecipientsList.svelte

```diff
@@ -0,0 +1,21 @@
+<!-- Send Recipients Component
+
+    The list of the recipients displayed on the end pages of the website
+
+    Author: Parul Garg
+    Last Modified: 10/08/2024
+-->
+
+<script>
+    import { ContactsStore } from "../stores/contacts-store"
+
+</script>
+  
+<div class="flex pb-8 flex-wrap justify-center self-center max-w-xl gap-2">
```

Comment:
In the future make sure to comment what the styling was for. eg which section

Reply:
Done!